### DISCUSSION TITLE
Godot 4 - Tilemap Bitmask editing - Add Color Blind mode.

### DISCUSSION THREAD

**Discussion by percybolmer** [godotengine/godot-proposals#6077] [GUI]
In Godot 3, it is easy to spot the Bitmask when editing. Its a clear red color with a dark background.

![image](https://user-images.githubusercontent.com/7509921/211496478-0d818fa2-a84f-40d8-b582-f30c7f09d40f.png)

In Godot 4, as a color-blind I find this very hard to see. Seems the background is brighter and so is the Red color.
Maybe an option to enhance the Color? In godot 3 it's a bit redder and easier for me to spot.

![image](https://user-images.githubusercontent.com/7509921/211496275-ace56315-c983-489d-89a7-6d56d4d84458.png)

**Comment by AThousandShips** [comment]
Note that the bitmask is based on the terrain colour so you can change it
